prime-angular-frontend/src/app/core/resources/site-resource.service.ts

```diff
@@ -385,9 +385,22 @@ export class SiteResource {
       );
   }
 
+  public enableEditingSite(siteId: number): Observable<Site> {
+    return this.apiResource.put<Site>(`sites/${siteId}/enable-editing`)
+      .pipe(
+        map((response: ApiHttpResponse<Site>) => response.result),
+        tap(() => this.toastService.openSuccessToast('Site registration has been enabled editing')),
```

Comment:
I think all of these messages should have the format of "Site registration editing has been enabled" or "Site registration editing could not be enabled".

Reply:
Fixed in 5973e27